nautobot/ui/src/constants/apiPath.js

```diff
@@ -1,8 +1,10 @@
 export const API_BASE = "/api";
-export const API_UI_BASE = "/ui";
 
-export const API_USER_SESSION_INFO = "/users/users/session";
+export const API_USER_SESSION_INFO = `${API_BASE}/users/users/session/`;
+export const API_USER_AUTHENTICATE = `${API_BASE}/users/tokens/authenticate/`;
 
-export const API_UI_MENU_INFO = "/ui/get-menu";
+export const API_UI_BASE = `${API_BASE}/ui`;
+export const API_UI_MENU_INFO = `${API_UI_BASE}/get-menu/`;
+export const API_OBJECT_COUNTS = `${API_UI_BASE}/get-object-counts/`;
```

Comment:
I'm missing something here - previously `API_UI_MENU_INFO` was `/ui/get-menu/` but now it's `/api/ui/get-menu/` - what's the corresponding change to the usage of this constant?

Reply:
I need RTK Query to be able to access `/logout` (no `/api` prefix) and likely `/login` because they have a larger impact when it comes to SSO/LDAP, etc. So instead of having two RTK Query objects (complicates `invalidatesTags`) I updated this line:

https://github.com/nautobot/nautobot/pull/3744/files#diff-b5b58239f7ec89457879d5636cad867bbb87d6ebc3179f15c58513c7c98f2ca2R20